src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestWebServerStartup.cs

```diff
@@ -0,0 +1,27 @@
+using System.IO;
```

Comment:
this needs the copyright header:

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using System.IO;
```